specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Unresolvable_Import.cs

```diff
@@ -0,0 +1,20 @@
+using DotNetProjectFile.MsBuild;
+
+namespace Rules.MS_Build.Unresolvable_Import;
+
+public class Reports
+{
+    [Test]
+    public void empty_includes() => new UnresolvableImport()
+    .ForProject("UnresolvableImport.cs")
+        .HasIssue(Issue.WRN("Proj0034", "The <Import> '$(MSBuildThisFileDirectory)common.props' could not be resolved.").WithSpan(02, 02, 02, 62));
```

Comment:
Perhaps this warning message gives the false impression that the import is incorrectly defined, rather than the analyzers simply not knowing how to handle it.

Reply:
Suggestion how to improve on that?

Reply:
I added "... by the analyzer."